packages/react-components/src/templates/__tests__/EventPage.test.tsx

```diff
@@ -45,8 +45,8 @@ it('renders the join event section', () => {
 });
 
 it('renders the event notes', () => {
-  const { getByText } = render(<EventPage {...props} notes="My Notes" />);
-  expect(getByText('My Notes')).toBeVisible();
+  const { getByRole } = render(<EventPage {...props} notes="My Notes" />);
+  expect(getByRole('heading', { name: 'Notes', level: 2 })).toBeVisible();
```

Comment:
What is `name`? o.O

Reply:
sneaky way to differentiate between headings